src/loaddb/load_server_loader.cpp

```diff
@@ -325,18 +326,24 @@ namespace cubload
     const std::vector<std::string> &classes_ignored = m_session.get_args ().ignore_classes;
     bool is_ignored;
 
-    char lower_case_string[DB_MAX_IDENTIFIER_LENGTH] = { 0 };
-    std::string class_name (lower_case_string);
-
-    assert (intl_identifier_lower_string_size (class_name.c_str ()) <= DB_MAX_IDENTIFIER_LENGTH);
+    char *lower_case_string = (char *) db_private_alloc (NULL, intl_identifier_lower_string_size (classname),
```

Comment:
- size probably needs `+ 1` for terminator character
- is last argument correct (`sizeof (char)`)?

Reply:
You are right. :) Fixing right away.